documentation-website/src/components/navbar.tsx

```diff
@@ -49,6 +49,12 @@ const Navbar = ({
             label={'LINKEDIN'}
           />
         </li>
+        <li>
+          <ExternalLink
+            to="https://www.https://idrinth-api-bench.slack.com/join/shared_invite/zt-2f4zmw2sz-c3etHzCFq3LtZpkR15xXMA#/shared-invite/email.com/groups/9588634/"
```

Comment:
that link is incorrect, it starts with `https://www.` before having the right ``https://`